src/pages/home/report/ReportActionItem.js

```diff
@@ -90,6 +94,14 @@ class ReportActionItem extends Component {
         this.textInput.focus();
     }
 
+    setHovered() {
+        this.setState({hovered: true});
+    }
+
+    unsetHovered() {
+        this.setState({hovered: false});
+    }
```

Comment:
This should happen in `Hoverable` which can return the `hovered` state as it was doing before.

Reply:
Sure.